README.md

```diff
@@ -1,7 +1,25 @@
 [![linting: pylint](https://img.shields.io/badge/linting-pylint-yellowgreen)](https://github.com/pylint-dev/pylint)
 [![mypy](https://img.shields.io/badge/mypy-checked-brightgreen.svg)](https://mypy.readthedocs.io/en/stable/)
-# Setup
+[![PyPI version](https://badge.fury.io/py/Django.svg)](https://badge.fury.io/py/Django)
+[![PyPI version](https://badge.fury.io/py/crispy-bootstrap4.svg)](https://badge.fury.io/py/crispy-bootstrap4)
+[![PyPI version](https://badge.fury.io/py/django-crispy-forms.svg)](https://badge.fury.io/py/django-crispy-forms)
+[![PyPI version](https://badge.fury.io/py/social-auth-app-django.svg)](https://badge.fury.io/py/social-auth-app-django)
```

Comment:
Where did you get them?

Reply:
Should it look like this?
![image](https://github.com/Strexas/qr_web/assets/75167873/d044c47b-0f6e-4320-9908-f71216dd23c9)

Reply:
Also not part of your task

Reply:
https://badge.fury.io/for/py

Reply:
Our project is not PyPi module (PyPi is what you install with `pip install`), if you want to add badge search for official ones. Here is for Django
https://www.djangoproject.com/community/badges/

Reply:
Or specify name of PyPi package